README.md

```diff
@@ -4,6 +4,15 @@ A project based learning activity for people who are getting started with Git an
 
 You can play the game at: https://githubschool.github.io/github-games/
```

Comment:
You can play the game at: https://githubschool.github.io/github-games/
You can play the game [here]: (https://githubschool.github.io/github-games/)